src/main/java/core/basesyntax/report/CsvReportCreator.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.report;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Map;
+
+public class CsvReportCreator implements ReportCreator {
+    private String filePath;
+
+    public CsvReportCreator(String filePath) {
+        this.filePath = filePath;
+    }
+
+    @Override
+    public void createReport(Map<String, Integer> fruitInventory, String filePath) {
+        try (FileWriter writer = new FileWriter(filePath)) {
+            writer.write("fruit,quantity\n");
```

Comment:
All magic numbers, strings, and symbols in your code should be constants.
[this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)
```suggestion
            writer.write(HEADER);
```